controllers/control-plane/kcp_manifest_test.go

```diff
@@ -5,31 +5,19 @@ import (
 	. "github.com/kyma-project/lifecycle-manager/pkg/testutils"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"sigs.k8s.io/controller-runtime/pkg/envtest"
 )
 
 var _ = Describe("Manifest.Spec.Remote in KCP mode", Ordered, func() {
 	kyma := NewTestKyma("kyma")
+	kyma.Labels[v1beta2.SyncLabel] = v1beta2.DisableLabelValue
 
-	module := v1beta2.Module{
-		ControllerName: "manifest",
-		Name:           NewUniqModuleName(),
-		Channel:        v1beta2.DefaultChannel,
-	}
+	module := NewTestModule("module", v1beta2.DefaultChannel)
 	kyma.Spec.Modules = append(kyma.Spec.Modules, module)
-	var runtimeEnv *envtest.Environment
-	BeforeAll(func() {
```

Comment:
To Reviewer:

For this test, we don't need to provision runtimeEnv, it will increase test processing time.